tests/tests-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+message(STATUS "Building obelisk_cpp tests")
+
+# Catch2 for testing
+include(FetchContent)
+
+FetchContent_Declare(
+        Catch2
+        GIT_REPOSITORY https://github.com/catchorg/Catch2.git
+        GIT_TAG        v3.5.4 # or a later release
```

Comment:
use spaces and not tabs (we should probably add this to the style checking for c++ code)

Reply:
this is formatting in the cmake - tbh not entirely sure how to do that right now. I can take a look though